### PR TITLE
Add comment with more description of the new experiment platform labels

### PR DESCRIPTION
**Pull request by lmorchard**
Trying this again to trigger a review auto-request

### CODE REVIEW

**Comment by lmorchard** [comment]
Ah hah! Looks like it worked this time

File: locales/en-US/app.ftl [hunk 1]
```diff
@@ -150,14 +150,17 @@ userCountContainerAlt = Just launched!
...
+[[experimentPlatform]]
 experimentPlatformWebAddonMobile = Firefox / web / mobile experiment
 experimentPlatformWebAddon = Firefox / web experiment
 experimentPlatformWebMobile = web / mobile experiment
 experimentPlatformAddonMobile = Firefox / mobile experiment
 experimentPlatformWeb = web experiment
 experimentPlatformAddon = Firefox experiment
-experimentPlatformMobile = Available on mobile
-experimentGoToLink = Go to { $title }
+experimentPlatformMobile = mobile experiment
```
Comment (flodolo) [inline locales/en-US/app.ftl]: At this point we already have 7/8 languages in. Are there technical limitations preventing to change this string ID? If there aren't, we sadly need to.

**Review by flodolo — CHANGES_REQUESTED** [review]

File: locales/en-US/app.ftl (hunk 1 above)
Comment (lmorchard) [reply locales/en-US/app.ftl]: Just pushed an update that changes the string ID. Looks like that was part of the copy/paste from the string changes pre-merge that I had uncommitted :/

**Review by flodolo — APPROVED** [review]
Thanks, this works.

**Pull request merged**